Code/GraphMol/MolEnumerator/Wrap/rough_test.py

```diff
@@ -130,6 +131,30 @@ def testCombined(self):
     bndl = rdMolEnumerator.Enumerate(m)
     self.assertEqual(bndl.Size(), 9)
 
+    def testIssue3231(self):
+      mol = Chem.MolFromSmiles(
+        'C[C@H](OC1=C(N)N=CC(C2=CN(C3C[C@H](C)NCC3)N=C2)=C1)C4=C(Cl)C=CC(F)=C4Cl')
+      Chem.AssignStereochemistry(mol, force=True, flagPossibleStereoCenters=True)
+      l = Chem.FindMolChiralCenters(mol, includeUnassigned=True)
+      self.assertEqual(l, [(1, 'S'), (12, '?'), (14, 'S')])
+
+      enumsi_opt = rdMolEnumerator.StereoEnumerationOptionsV2(maxIsomers=20, onlyUnassigned=False)
+      isomers = list(rdMolEnumerator.EnumerateStereoisomersV2(mol, enumsi_opt))
+      chi_cents = []
+      for iso in isomers:
+        Chem.AssignStereochemistry(iso)
+        chi_cents.append(Chem.FindMolChiralCenters(iso, includeUnassigned=True))
+
+      self.assertEqual(sorted(chi_cents),
```

Comment:
I know this is annoying, but would you mind making the indentation more coherent here? If clang-format is messing it up, this is a reasonable place to use clang-format: off

Reply:
In this particular case: we use `yapf` for python code, not clang-format